library/model/book.py

```diff
@@ -155,12 +138,10 @@ def renew_rental(self) -> "BorrowedBook":
         self.current_fee += self.get_weekly_fee()
         return self
 
-    def return_book(self) -> Book:
+    def return_book_to_library(self):
```

Comment:
current code can not update library,need to add the following two lines:
 LibraryRepository.update_book(self) 
 return self